src/stats.md

```diff
@@ -151,10 +156,25 @@ graph than other graphs: rather than changing which cards or period of
 studying is included, it limits how far out the intervals are displayed
 to (so 14-month intervals are not displayed at all on a 1-year graph).
 
-**Card Ease**
+**Card Ease**\
 This graph shows the number of cards that have a specific ease factor.
 Average ease of the selected deck / collection is also displayed here.
 
+**Card Stability**\
+Only shown if FSRS is enabled. Stability is defined as the amount of time required
+for the probability of recall to decrease from 100% to 90%.
+
+**Card Difficulty**\
+Only shown if FSRS is enabled. Difficulty determines how quickly the intervals grow
+after each review.
+
+**Card Retrievability**\
+Only shown if FSRS is enabled. "Retrievability" is synonymous with "probability of recall".
+
+Estimated total knowledge is the number of cards the user is expected to know today, 
```

Comment:
Expected can read as a requirement ("you are expected to attend the class"), so how about the following instead?

```suggestion
Estimated total knowledge is the number of cards the user is likely to know today, 
```

Other suggestions welcome.

Reply:
I prefer expected in this case. @brishtibheja thoughts?

Reply:
How about 'predicted'?

Reply:
"Predicted to know today" sounds odd to me, granted I'm not a native English speaker

Reply:
Sounds fine to me, but happy for someone else to weigh in.

Reply:
I think we should use "you" instead of "the user".

As for the verb, "likely" sounds best for the reason dae mentioned. But if we were to use "expected", how about: 
```suggestion
Estimated total knowledge is the number of cards you can expect to know today, 
```

I would still prefer to use:
```suggestion
Estimated total knowledge is the number of cards you are likely to know today, 
```

Reply:
I'll wait for Dae

Reply:
Mild preference for 'likely'. One other thought while we're bike-shedding: to avoid any confusion with cards that happen to be due today, one other idea would be something like:

Estimated total knowledge is the number of cards you are likely to currently remember.

Reply:
Ok, check the latest commit